Build-Toolkit-Components.ps1

```diff
@@ -104,6 +104,20 @@ if ($null -eq $ExcludeMultiTargets)
     $ExcludeMultiTargets = @()
 }
 
+# Both uwp and wasdk share a targetframework. Both cannot be enabled at once.
+# If both are supplied, remove one based on WinUIMajorVersion.
+if ($MultiTargets.Contains('uwp') -and $MultiTargets.Contains('wasdk'))
+{
+    if ($WinUIMajorVersion -eq 2)
+    {
+        $ExcludeMultiTargets = $ExcludeMultiTargets + 'wasdk'
+    }
+    else
+    {
+        $ExcludeMultiTargets = $ExcludeMultiTargets + 'uwp'
+    }
+}
+
```

Comment:
Seems like we're doing this in 3 places, should we have a helper function and start making another helper script these all include and reference? (maybe https://learn.microsoft.com/en-us/powershell/scripting/learn/ps101/10-script-modules ? )

Reply:
There are chunks of these files with overlapping functionality, not just this. For now, the code should be largely aligned to make it easier to extract later. We can file a ticket for it, if you like?